<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tasks</class>
 <widget class="QDialog" name="tasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Εργασίες</string>
  </property>
  <widget class="QComboBox" name="comboCustomer">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeStart">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yy h:mm AP</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeStop">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>d/M/yy h:mm AP</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboPlace">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboProject">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>251</width>
     <height>78</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Πελάτης</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Τόπος</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Έργο</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Έναρξη</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Λήξη</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineDebit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Χρέωση/ώρα</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
